webpack/scenes/Subscriptions/SubscriptionActions.js

```diff
@@ -126,7 +127,9 @@ export const deleteSubscriptions = poolIds => async (dispatch) => {
   };
 
   try {
+    const POLL_TASK_INTERVAL = 5000;
```

Comment:
Is it possible / does it make sense to move this constant into TaskActions so we can use it in multiple places?

Reply:
Also looks like SubcriptionsPage.js has the same constant defined, so it could potentially be reused there too

Reply:
I am putting it in `TaskConstants.js ` because this is being used by manifest tasks and also by subscription task.